cpu/native/periph/uart.c

```diff
@@ -135,7 +135,6 @@ int uart_init(uart_t uart, uint32_t baudrate, uart_rx_cb_t rx_cb, void *arg)
     case 57600: speed = B57600; break;
     case 115200: speed = B115200; break;
     case 230400: speed = B230400; break;
-#if __linux__
```

Comment:
Some of these baudrates were already included unconditionally (e.g. 115200) so I think it's safe to assume they are available everywhere.